@here/harp-mapview/test/VisibleTileSetTest.ts

```diff
@@ -47,7 +47,8 @@ describe("VisibleTileSet", function() {
 
         ds.attach(new FakeMapView() as MapView);
 
-        const renderList = vts.updateRenderList(worldCenter, zoomLevel, storageLevel, [ds]);
+        vts.updateRenderList(worldCenter, zoomLevel, storageLevel, [ds]);
+        const renderList = vts.dataSourceTileList;
```

Comment:
I think we should rename dataSourceTileList to renderList as well, to be consistent.

Reply:
Done